pkg/tsdb/azuremonitor/azure-resource-graph-datasource.go

```diff
@@ -236,8 +236,45 @@ func (e *AzureResourceGraphDatasource) unmarshalResponse(res *http.Response) (Az
 	}()
 
 	if res.StatusCode/100 != 2 {
+		type ErrorObject struct {
+			Code    string `json:"code"`
+			Message string `json:"message"`
+			Details []struct {
```

Comment:
It's a recursive list of the same `ErrorObject`

Look the error schema here: [Microsoft REST API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses)